specification/security/_types/RoleDescriptor.ts

```diff
@@ -52,6 +52,7 @@ export class RoleDescriptor {
    * @doc_id run-as-privilege
    */
   run_as?: string[]
+  description?: string
```

Comment:
This is not related to the new API.
Role Descriptors have a description field since https://github.com/elastic/elasticsearch/pull/107088 .